dashboard/views.py

```diff
@@ -480,7 +480,7 @@ def assignments(request, course_id=0):
 def get_course_assignments(course_id):
 
     sql=f"""select assign.*,sub.avg_score from
-            (select assignment_id,name,assign_grp_name,grp_id,due_date,points_possible,group_points,weight,drop_lowest,drop_highest from
+            (select ifnull(assignment_id, 0) as assignment_id ,name,assign_grp_name,grp_id,due_date,points_possible,group_points,weight,drop_lowest,drop_highest from
```

Comment:
Since Id's are now integers and in case of hidden assignments in the weighted assignment group the assignment id is null and pandas defaults the type to float and when doing the join all the assignment id's are changed. More [info](https://github.com/tl-its-umich-edu/my-learning-analytics/issues/578#issuecomment-512793409)